modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -108,7 +108,11 @@ public SELF withConnectTimeoutSeconds(int connectTimeoutSeconds) {
     }
 
     public SELF withInitScript(String initScriptPath) {
-        this.initScriptPath = initScriptPath;
+        return withMultiInitScript(initScriptPath);
+    }
+
+    public SELF withMultiInitScript(String... initScriptPaths) {
```

Comment:
Just a thought but... If the parameters were `String, String...` couldn't we keep the original method name and avoid problems with overloading/binary incompatibility? 🤔

Reply:
I had the same thought, but assumed the purpose was to keep the API visually and functionally the same. Now that you raise it up, it is a good question. The previous iteration was a lot simpler and less complicated.